docs/core/testing/mstest-analyzers/overview.md

```diff
@@ -15,6 +15,42 @@ ms.date: 12/20/2023
 
 The rules are organized into categories such as performance usage...
 
+Starting with MSTest.TestFramework 3.7, the MSTest.Analyzers NuGet package is a dependency of the framework. For earlier versions, you need to use `MSTest` metapackage or add a package reference for `MSTest.Analyzers` explicitly.
+
+## MSTestAnalysisMode
+
+Starting with MSTest 3.8, an MSBuild property named `MSTestAnalysisMode` is available to determine which analyzers are enabled at which severity.
+
+The available values are:
+
+### `None`
+
+This sets all analyzers to `none` severity, making all of them disabled. You can then enable individual analyzers using `.editorconfig` or `.globalconfig` files.
+
+### `Default`
+
+This follows the default documented behavior for each rule.
+
+- Rules that are enabled by default will use their default severity.
+- Rules that are disabled by default will use `none` severity.
+
+### `Recommended`
```

Comment:
It might be a bit too boring to maintain the doc but it would probably be awesome if we could list here the rules enabled by this mode.

Reply:
Maybe we could generate a markdown per version and we would include it here.
@IEvangelist is this something that is common?

Reply:
@gewarren what's your opinion here?

Reply:
Given we have NuGet not part of SDK or VS, it'd be more complex for us. But I think just pointing out to the file in our NUGet package is a good idea.

Reply:
@gewarren @Evangelink I added some info on how to locate the globalconfig inside the NuGet package. Let me know if it looks good